common/primitives/src/messages.rs

```diff
@@ -130,6 +139,43 @@ impl<T> BlockPaginationResponse<T> {
 	}
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+/// A trait for helping setup state for running benchmarks.
+pub trait BenchmarkHelper<AccountId> {
```

Comment:
I would love to see more explanation here about what this helper does and how to use it with benchmarks.  Maybe also link to the StackOverflow/StackExchange question.

Reply:
Me too